src/applications/travel-pay/tests/components/submit-flow/pages/CantFilePage.unit.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import { $ } from 'platform/forms-system/src/js/utilities/ui';
+
+import CantFilePage from '../../../../components/submit-flow/pages/CantFilePage';
```

Comment:
I suppose I should have caught this earlier, but what about `CannotFilePage` to make it more readable?

Reply:
I recommend `UnsupportedClaimTypePage`